pylint_odoo/checkers/no_modules.py

```diff
@@ -359,6 +399,15 @@ def visit_call(self, node):
                         # The argument "selection_add" is for overwrite field.
                         # Then don't need help.
                         has_help = None
+                    elif argument.arg in \
```

Comment:
Avoid use `\`
You could use a `elif argument.arg in (\nself.config.deprecated_field_parameters):` instead.